model/src/main/java/com/ge/predix/acs/rest/PolicyEvaluationRequestV1.java

```diff
@@ -43,6 +41,10 @@
 
     private LinkedHashSet<String> policySetsEvaluationOrder = EMPTY_POLICY_EVALUATION_ORDER;
 
+    public static LinkedHashSet<String> getEmptyPolicyEvaluationOrder() {
+        return EMPTY_POLICY_EVALUATION_ORDER;
```

Comment:
we should probably revert this change and the one above since i'm removing LinkedHashSet in favor of the Set interface in another feature branch